client/my-sites/plugins/plugin-settings-link/index.jsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+
+export default React.createClass( {
+	displayName: 'PluginSettingsLink',
+
+	propTypes: {
+		link: React.PropTypes.string.isRequired,
```

Comment:
Will try to do this from now on with the PropTypes. However I do like it when things are a bit more explicit.